### PR TITLE
CWC19 Group Stage update

### PR DESCRIPTION
**Pull request by juankristal**

### CODE REVIEW

File: wiki/Tournaments/CWC/2019/en.md [hunk 1]
```diff
@@ -89,9 +89,87 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| Italy | ![][flag_IT] | ![][flag_CL] | Chile | **16:00 UTC** | 18:00 UTC+02 | 12:00 UTC-04 |
+| Venezuela | ![][flag_GB] | ![][flag_PL] | Poland | **16:00 UTC** | 12:00 UTC-04 | 18:00 UTC+02 |
+| Colombia | ![][flag_CO] | ![][flag_FR] | France | **16:00 UTC** | 12:00 UTC-04 | 18:00 UTC+02 |
+
+---
+
 ## Mappools
 
-_Mappools won't be available until one week before the stage that it is meant to be played_
+[Download the map pack here!](https://www.dropbox.com/sh/e3jgd039od90c85/AADfBfWbrPI5NHwpKWiv3fCCa?dl=0)
```
Comment (fajar13k) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
**[Download the map pack here!](https://www.dropbox.com/sh/e3jgd039od90c85/AADfBfWbrPI5NHwpKWiv3fCCa?dl=0)**
```
Bold

File: wiki/Tournaments/CWC/2019/en.md [hunk 2]
```diff
@@ -89,9 +89,87 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| Mexico | ![][flag_MX] | ![][flag_CL] | Chile | **18:00 UTC** | 13:00 UTC-05 | 14:00 UTC-04 |
+| Canada | ![][flag_CA] | ![][flag_CL] | Chile | **20:00 UTC** | 16:00 UTC-04 | 16:00 UTC-04 |
+| Sweden | ![][flag_SE] | ![][flag_CO] | Colombia | **20:00 UTC** | 22:00 UTC+02 | 16:00 UTC-04 |
+| Brazil | ![][flag_BR] | ![][flag_GB] | United States | **21:00 UTC** | 18:00 UTC-03 | 17:00 UTC-04 |
+| Canada | ![][flag_CA] | ![][flag_MX] | Mexico | **21:00 UTC** | 17:00 UTC-04 | 16:00 UTC-05 |
+
+### Sunday, 30. June 2018
+
+| Team A |  |  | Team B | Match Time (UTC) | Local Time A | Local Time B | |
+| ---: | ---: | :--- | :--- | :---: | :---: | :---: | :---: |
```
Comment (fajar13k) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| --: | --: | :-- | :-- | :-: | :-: | :-: | :-: |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 3]
```diff
@@ -89,9 +89,87 @@ The osu!catch World Cup 2019 is run by various community members by distributing
 
 ---
 
+## Match Schedule: Group Stage
+
+### Saturday, 29. June 2018
+
+| Team A |  |  | Team B | Match Time (UTC) | Local Time A | Local Time B | |
+| ---: | ---: | :--- | :--- | :---: | :---: | :---: | :---: |
```
Comment (fajar13k) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| --: | --: | :-- | :-- | :-: | :-: | :-: | :-: |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 4]
```diff
@@ -89,9 +89,87 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| Romania | ![][flag_RO] | ![][flag_GB] | United States | **16:00 UTC** | 18:00 UTC+02 | 12:00 UTC-04 |
+| Venezuela | ![][flag_GB] | ![][flag_FI] | Finland | **17:00 UTC** | 13:00 UTC-04 | 20:00 UTC+03 |
+| Italy | ![][flag_IT] | ![][flag_MX] | Mexico | **17:00 UTC** | 19:00 UTC+02 | 12:00 UTC-05 |
+| Russian Federation | ![][flag_RU] | ![][flag_GB] | United States | **17:00 UTC** | 22:00 UTC+05 | 13:00 UTC-04 |
+| Belgium | ![][flag_BE] | ![][flag_GB] | Venezuela | **18:00 UTC** | 19:00 UTC+01 | 14:00 UTC-04 |
+| Canada | ![][flag_CA] | ![][flag_IT] | Italy | **18:00 UTC** | 14:00 UTC-04 | 20:00 UTC+02 |
+| Mexico | ![][flag_MX] | ![][flag_CL] | Chile | **18:00 UTC** | 13:00 UTC-05 | 14:00 UTC-04 |
+| Canada | ![][flag_CA] | ![][flag_CL] | Chile | **20:00 UTC** | 16:00 UTC-04 | 16:00 UTC-04 |
+| Sweden | ![][flag_SE] | ![][flag_CO] | Colombia | **20:00 UTC** | 22:00 UTC+02 | 16:00 UTC-04 |
+| Brazil | ![][flag_BR] | ![][flag_GB] | United States | **21:00 UTC** | 18:00 UTC-03 | 17:00 UTC-04 |
```
Comment (fajar13k) [inline wiki/Tournaments/CWC/2019/en.md]: GB but United States? I don't know which is correct
More occurences above and below, some are in Vietnam. (check line 109)

File: wiki/Tournaments/CWC/2019/en.md [hunk 5]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| Hong Kong | ![][flag_HK] | ![][flag_KR] | South Korea | **11:00 UTC** | 19:00 UTC+08 | 20:00 UTC+09 |
+| Japan | ![][flag_JP] | ![][flag_CN] | China | **11:00 UTC** | 20:00 UTC+09 | 19:00 UTC+08 |
+| Sweden | ![][flag_SE] | ![][flag_VN] | Vietnam | **11:00 UTC** | 13:00 UTC+02 | 18:00 UTC+07 |
+| Taiwan | ![][flag_TW] | ![][flag_PH] | Philippines | **12:00 UTC** | 20:00 UTC+08 | 20:00 UTC+08 |
+| Netherlands | ![][flag_NL] | ![][flag_CN] | China | **12:00 UTC** | 14:00 UTC+02 | 20:00 UTC+08 |
+| Japan | ![][flag_JP] | ![][flag_AU] | Australia | **12:00 UTC** | 21:00 UTC+09 | 22:00 UTC+10 |
+| Czech Republic | ![][flag_CZ] | ![][flag_NO] | Norway | **13:00 UTC** | 14:00 UTC+01 | 15:00 UTC+02 |
+| Australia | ![][flag_AU] | ![][flag_CN] | China | **13:00 UTC** | 23:00 UTC+10 | 21:00 UTC+08 |
+| Norway | ![][flag_NO] | ![][flag_TW] | Taiwan | **14:00 UTC** | 16:00 UTC+02 | 22:00 UTC+08 |
+| Belgium | ![][flag_BE] | ![][flag_FI] | Finland | **14:00 UTC** | 15:00 UTC+01 | 17:00 UTC+03 |
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| Belgium | ![][flag_BE] | ![][flag_FI] | Finland | **14:00 UTC** | 16:00 UTC+02 | 17:00 UTC+03 |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 6]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| Czech Republic | ![][flag_CZ] | ![][flag_PH] | Philippines | **14:00 UTC** | 15:00 UTC+01 | 22:00 UTC+08 |
+| Colombia | ![][flag_CO] | ![][flag_VN] | Vietnam | **14:00 UTC** | 9:00 UTC-05 | 21:00 UTC+07 |
+| Brazil | ![][flag_BR] | ![][flag_RU] | Russian Federation | **16:00 UTC** | 13:00 UTC-03 | 21:00 UTC+05 |
+| Romania | ![][flag_RO] | ![][flag_US] | United States | **16:00 UTC** | 18:00 UTC+02 | 12:00 UTC-04 |
+| Denmark | ![][flag_DK] | ![][flag_AR] | Argentina | **16:00 UTC** | 17:00 UTC+01 | 13:00 UTC-03 |
+| United Kingdom | ![][flag_GB] | ![][flag_DE] | Germany | **16:00 UTC** | 17:00 UTC+01 | 18:00 UTC+02 |
+| Venezuela | ![][flag_VE] | ![][flag_FI] | Finland | **17:00 UTC** | 13:00 UTC-04 | 20:00 UTC+03 |
+| Italy | ![][flag_IT] | ![][flag_MX] | Mexico | **17:00 UTC** | 19:00 UTC+02 | 12:00 UTC-05 |
+| Russian Federation | ![][flag_RU] | ![][flag_US] | United States | **17:00 UTC** | 22:00 UTC+05 | 13:00 UTC-04 |
+| Belgium | ![][flag_BE] | ![][flag_VE] | Venezuela | **18:00 UTC** | 19:00 UTC+01 | 14:00 UTC-04 |
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| Belgium | ![][flag_BE] | ![][flag_VE] | Venezuela | **18:00 UTC** | 20:00 UTC+02 | 14:00 UTC-04 |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 7]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| Japan | ![][flag_JP] | ![][flag_NL] | Netherlands | **11:00 UTC** | 20:00 UTC+09 | 13:00 UTC+02 |
+| Czech Republic | ![][flag_CZ] | ![][flag_TW] | Taiwan | **11:00 UTC** | 12:00 UTC+01 | 19:00 UTC+08 |
+| Thailand | ![][flag_TH] | ![][flag_ID] | Indonesia | **11:00 UTC** | 18:00 UTC+07 | 18:00 UTC+07 |
+| Norway | ![][flag_NO] | ![][flag_PH] | Philippines | **12:00 UTC** | 14:00 UTC+02 | 20:00 UTC+08 |
+| Thailand | ![][flag_TH] | ![][flag_HK] | Hong Kong | **12:00 UTC** | 19:00 UTC+07 | 20:00 UTC+08 |
+| Indonesia | ![][flag_ID] | ![][flag_KR] | South Korea | **12:00 UTC** | 19:00 UTC+07 | 21:00 UTC+09 |
+| Poland | ![][flag_PL] | ![][flag_FI] | Finland | **13:00 UTC** | 15:00 UTC+02 | 16:00 UTC+03 |
+| Russian Federation | ![][flag_RU] | ![][flag_RO] | Romania | **13:00 UTC** | 18:00 UTC+05 | 15:00 UTC+02 |
+| Vietnam | ![][flag_VN] | ![][flag_FR] | France | **13:00 UTC** | 20:00 UTC+07 | 15:00 UTC+02 |
+| Belgium | ![][flag_BE] | ![][flag_PL] | Poland | **15:00 UTC** | 16:00 UTC+01 | 17:00 UTC+02 |
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| Belgium | ![][flag_BE] | ![][flag_PL] | Poland | **15:00 UTC** | 17:00 UTC+02 | 17:00 UTC+02 |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 8]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| Japan | ![][flag_JP] | ![][flag_AU] | Australia | **12:00 UTC** | 21:00 UTC+09 | 22:00 UTC+10 |
+| Czech Republic | ![][flag_CZ] | ![][flag_NO] | Norway | **13:00 UTC** | 14:00 UTC+01 | 15:00 UTC+02 |
+| Australia | ![][flag_AU] | ![][flag_CN] | China | **13:00 UTC** | 23:00 UTC+10 | 21:00 UTC+08 |
+| Norway | ![][flag_NO] | ![][flag_TW] | Taiwan | **14:00 UTC** | 16:00 UTC+02 | 22:00 UTC+08 |
+| Belgium | ![][flag_BE] | ![][flag_FI] | Finland | **14:00 UTC** | 15:00 UTC+01 | 17:00 UTC+03 |
+| Czech Republic | ![][flag_CZ] | ![][flag_PH] | Philippines | **14:00 UTC** | 15:00 UTC+01 | 22:00 UTC+08 |
+| Colombia | ![][flag_CO] | ![][flag_VN] | Vietnam | **14:00 UTC** | 9:00 UTC-05 | 21:00 UTC+07 |
+| Brazil | ![][flag_BR] | ![][flag_RU] | Russian Federation | **16:00 UTC** | 13:00 UTC-03 | 21:00 UTC+05 |
+| Romania | ![][flag_RO] | ![][flag_US] | United States | **16:00 UTC** | 18:00 UTC+02 | 12:00 UTC-04 |
+| Denmark | ![][flag_DK] | ![][flag_AR] | Argentina | **16:00 UTC** | 17:00 UTC+01 | 13:00 UTC-03 |
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| Denmark | ![][flag_DK] | ![][flag_AR] | Argentina | **16:00 UTC** | 18:00 UTC+02 | 13:00 UTC-03 |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 9]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| United Kingdom | ![][flag_GB] | ![][flag_DE] | Germany | **16:00 UTC** | 17:00 UTC+01 | 18:00 UTC+02 |
+| Venezuela | ![][flag_VE] | ![][flag_FI] | Finland | **17:00 UTC** | 13:00 UTC-04 | 20:00 UTC+03 |
+| Italy | ![][flag_IT] | ![][flag_MX] | Mexico | **17:00 UTC** | 19:00 UTC+02 | 12:00 UTC-05 |
+| Russian Federation | ![][flag_RU] | ![][flag_US] | United States | **17:00 UTC** | 22:00 UTC+05 | 13:00 UTC-04 |
+| Belgium | ![][flag_BE] | ![][flag_VE] | Venezuela | **18:00 UTC** | 19:00 UTC+01 | 14:00 UTC-04 |
+| Canada | ![][flag_CA] | ![][flag_IT] | Italy | **18:00 UTC** | 14:00 UTC-04 | 20:00 UTC+02 |
+| Mexico | ![][flag_MX] | ![][flag_CL] | Chile | **18:00 UTC** | 13:00 UTC-05 | 14:00 UTC-04 |
+| Denmark | ![][flag_DK] | ![][flag_DE] | Germany | **18:00 UTC** | 19:00 UTC+01 | 20:00 UTC+02 |
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| Denmark | ![][flag_DK] | ![][flag_DE] | Germany | **18:00 UTC** | 20:00 UTC+02 | 20:00 UTC+02 |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 10]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| Poland | ![][flag_PL] | ![][flag_FI] | Finland | **13:00 UTC** | 15:00 UTC+02 | 16:00 UTC+03 |
+| Russian Federation | ![][flag_RU] | ![][flag_RO] | Romania | **13:00 UTC** | 18:00 UTC+05 | 15:00 UTC+02 |
+| Vietnam | ![][flag_VN] | ![][flag_FR] | France | **13:00 UTC** | 20:00 UTC+07 | 15:00 UTC+02 |
+| Belgium | ![][flag_BE] | ![][flag_PL] | Poland | **15:00 UTC** | 16:00 UTC+01 | 17:00 UTC+02 |
+| Brazil | ![][flag_BR] | ![][flag_RO] | Romania | **15:00 UTC** | 12:00 UTC-03 | 17:00 UTC+02 |
+| Sweden | ![][flag_SE] | ![][flag_FR] | France | **15:00 UTC** | 17:00 UTC+02 | 17:00 UTC+02 |
+| Italy | ![][flag_IT] | ![][flag_CL] | Chile | **16:00 UTC** | 18:00 UTC+02 | 12:00 UTC-04 |
+| Venezuela | ![][flag_VE] | ![][flag_PL] | Poland | **16:00 UTC** | 12:00 UTC-04 | 18:00 UTC+02 |
+| Colombia | ![][flag_CO] | ![][flag_FR] | France | **16:00 UTC** | 11:00 UTC-05 | 18:00 UTC+02 |
+| Denmark | ![][flag_DK] | ![][flag_GB] | United Kingdom | **17:00 UTC** | 18:00 UTC+01 | 18:00 UTC+01 |
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| Denmark | ![][flag_DK] | ![][flag_GB] | United Kingdom | **17:00 UTC** | 19:00 UTC+02 | 18:00 UTC+01 |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 11]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+
+| Team A |  |  | Team B | Match Time (UTC) | Local Time A | Local Time B | |
+| --: | --: | :-- | :-- | :-: | :-: | :-: | :-: |
+| Hong Kong | ![][flag_HK] | ![][flag_KR] | South Korea | **11:00 UTC** | 19:00 UTC+08 | 20:00 UTC+09 |
+| Japan | ![][flag_JP] | ![][flag_CN] | China | **11:00 UTC** | 20:00 UTC+09 | 19:00 UTC+08 |
+| Sweden | ![][flag_SE] | ![][flag_VN] | Vietnam | **11:00 UTC** | 13:00 UTC+02 | 18:00 UTC+07 |
+| Taiwan | ![][flag_TW] | ![][flag_PH] | Philippines | **12:00 UTC** | 20:00 UTC+08 | 20:00 UTC+08 |
+| Netherlands | ![][flag_NL] | ![][flag_CN] | China | **12:00 UTC** | 14:00 UTC+02 | 20:00 UTC+08 |
+| Japan | ![][flag_JP] | ![][flag_AU] | Australia | **12:00 UTC** | 21:00 UTC+09 | 22:00 UTC+10 |
+| Czech Republic | ![][flag_CZ] | ![][flag_NO] | Norway | **13:00 UTC** | 14:00 UTC+01 | 15:00 UTC+02 |
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| Czech Republic | ![][flag_CZ] | ![][flag_NO] | Norway | **13:00 UTC** | 15:00 UTC+02 | 15:00 UTC+02 |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 12]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| Australia | ![][flag_AU] | ![][flag_CN] | China | **13:00 UTC** | 23:00 UTC+10 | 21:00 UTC+08 |
+| Norway | ![][flag_NO] | ![][flag_TW] | Taiwan | **14:00 UTC** | 16:00 UTC+02 | 22:00 UTC+08 |
+| Belgium | ![][flag_BE] | ![][flag_FI] | Finland | **14:00 UTC** | 15:00 UTC+01 | 17:00 UTC+03 |
+| Czech Republic | ![][flag_CZ] | ![][flag_PH] | Philippines | **14:00 UTC** | 15:00 UTC+01 | 22:00 UTC+08 |
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| Czech Republic | ![][flag_CZ] | ![][flag_PH] | Philippines | **14:00 UTC** | 16:00 UTC+02 | 22:00 UTC+08 |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 13]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+
+### Sunday, 30. June 2018
+
+| Team A |  |  | Team B | Match Time (UTC) | Local Time A | Local Time B | |
+| --: | --: | :-- | :-- | :-: | :-: | :-: | :-: |
+| Australia | ![][flag_AU] | ![][flag_NL] | Netherlands | **10:00 UTC** | 20:00 UTC+10 | 12:00 UTC+02 |
+| Thailand | ![][flag_TH] | ![][flag_KR] | South Korea | **10:00 UTC** | 17:00 UTC+07 | 19:00 UTC+09 |
+| Hong Kong | ![][flag_HK] | ![][flag_ID] | Indonesia | **10:00 UTC** | 18:00 UTC+08 | 17:00 UTC+07 |
+| Japan | ![][flag_JP] | ![][flag_NL] | Netherlands | **11:00 UTC** | 20:00 UTC+09 | 13:00 UTC+02 |
+| Czech Republic | ![][flag_CZ] | ![][flag_TW] | Taiwan | **11:00 UTC** | 12:00 UTC+01 | 19:00 UTC+08 |
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
| Czech Republic | ![][flag_CZ] | ![][flag_TW] | Taiwan | **11:00 UTC** | 13:00 UTC+02 | 19:00 UTC+08 |
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 14]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
 
 ---
 
+## Match Schedule: Group Stage
+
+### Saturday, 29. June 2018
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
### Saturday, 29. June 2019
```

File: wiki/Tournaments/CWC/2019/en.md [hunk 15]
```diff
@@ -89,9 +89,93 @@ The osu!catch World Cup 2019 is run by various community members by distributing
...
+| Canada | ![][flag_CA] | ![][flag_IT] | Italy | **18:00 UTC** | 14:00 UTC-04 | 20:00 UTC+02 |
+| Mexico | ![][flag_MX] | ![][flag_CL] | Chile | **18:00 UTC** | 13:00 UTC-05 | 14:00 UTC-04 |
+| Denmark | ![][flag_DK] | ![][flag_DE] | Germany | **18:00 UTC** | 19:00 UTC+01 | 20:00 UTC+02 |
+| Argentina | ![][flag_AR] | ![][flag_GB] | United Kingdom | **18:00 UTC** | 15:00 UTC-03 | 19:00 UTC+01 |
+| Canada | ![][flag_CA] | ![][flag_CL] | Chile | **20:00 UTC** | 16:00 UTC-04 | 16:00 UTC-04 |
+| Sweden | ![][flag_SE] | ![][flag_CO] | Colombia | **20:00 UTC** | 22:00 UTC+02 | 15:00 UTC-05 |
+| Brazil | ![][flag_BR] | ![][flag_US] | United States | **21:00 UTC** | 18:00 UTC-03 | 17:00 UTC-04 |
+| Canada | ![][flag_CA] | ![][flag_MX] | Mexico | **21:00 UTC** | 17:00 UTC-04 | 16:00 UTC-05 |
+
+### Sunday, 30. June 2018
```
Comment (O1CDeif) [inline wiki/Tournaments/CWC/2019/en.md]:
```suggestion
### Sunday, 30. June 2019
```

**Pull request merged**